protocol/src/lib.rs

```diff
@@ -131,6 +134,20 @@ pub enum Message {
     Ruok,
     Imok,
 
+    /*
+     * Repair related
+     */
+    /// Close the given extent ID on the downstairs.
+    ExtentClose(u64, u64),
```

Comment:
There should be a comment here explaining what the arguments are - `rep_id`, not `job_id` like I originally assumed :)

Reply:
Commented in 9acce4f